Dockerfile

```diff
@@ -61,7 +61,11 @@ RUN apt update -y -q && apt upgrade -y -q && apt update -y -q && \
     re2c \
     perl \
     cpanminus \
-    openssh-client
+    openssh-client \ 
+    # used for CompCert Compilation
```

Comment:
maybe mention how many of these new deps are used for compcert? Not needed right now, but could be helpful if we add more dependencies afterwards.

Reply:
I will do this